### PR TITLE
Guardian Fixes/tweaks

### PR DESCRIPTION
**Pull request by Aurorablade**
Added in summoner missing check on recall manifest,

Am still trying to figure out why the admin item is not working if someone could help or suggest another way to do it.

### CODE REVIEW

**Comment by LufgtHuron** [comment]
These aren't guardian spirits are they..? I hope I didnt miss THAT addition because I do need to complain about it

**Comment by Earthdivine** [comment]
@LufgtHuron They are, the PR had been up for about a month and were just merged in yesterday.

File: code/modules/mob/living/simple_animal/guardian/guardian.dm [hunk 1]
```diff
@@ -633,6 +622,7 @@
 	if(random)
 		gaurdiantype = pick(possible_guardians)
 	if(adminseal)
+		world << "[user] [key]"
```
Comment (Fox-McCloud) [inline code/modules/mob/living/simple_animal/guardian/guardian.dm]: debugging?

**Comment by Fox-McCloud** [comment]
What is the admin item doing, exactly?

**Comment by Aurorablade** [comment]
Okay bear with me at work on phone and suck at explaining   The item goes as such for admin :admin spawns item, admin uses item and picks a player , admin is now players guardian. I could just make it an admin verb but I would need help setting up the verb proper. 

Also yea it is for devugging and I will add butcher stuff next carnivAl but I got another project before then

**Comment by Fox-McCloud** [comment]
What happens when the item is *actually* used?

**Comment by Aurorablade** [comment]
What's supposed to happen is the admin picks a player to guard what current happens as of testing on live server is nothing ABSOLUTELY NTHING  well I do get the list of players to

**Comment by Aurorablade** [comment]
DAMN PHONE

**Comment by Aurorablade** [comment]
As iwas sayingg of players to pick to Fuad but otherwise nothing after

**Comment by Fox-McCloud** [comment]
Right. Hm..

**Comment by Aurorablade** [comment]
okay heres how is...you can now spawn the seal guardian with your admin powers, find its summoner varible..with your admin powers..and set the summoner to a player in the world...POOF HAS GUARDIAN SEAL.

Unless someone has any other suggestions you can review this. There was a request to make the syndicate guardians refundable...but i am unsure on that code, i found the code for it for syndie borgs but was unsure the code would work/how would work for the standard PDA uplink type.

**Pull request merged**